web/src/main/kotlin/no/ssb/kostra/web/viewmodel/KostraErrorCode.kt

```diff
@@ -1,12 +0,0 @@
-package no.ssb.kostra.web.viewmodel
-
-import io.micronaut.core.annotation.Introspected
-
-@Introspected
-enum class KostraErrorCode {
-    NO_ERROR,
-    NORMAL_ERROR,
-    CRITICAL_ERROR,
-    SYSTEM_ERROR,
-    PARAMETER_ERROR
-}
```

Comment:
Vet ikke om denne kan slettes da man for de systemene som kaller API'et bør forholde seg til de samme returkodene som konsoll-app'en returnerer, dvs. 0 for NO_ERROR (ok), 1 for NORMAL_ERROR (advarsel) og 2 for CRITICAL_ERROR (error). Trenger en diskusjon her.